docs/release-notes/2.15.1.md

```diff
@@ -0,0 +1,30 @@
+# Percona Monitoring and Management 2.15.1
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">March 18, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Bugs Fixed
+
+* [PMM-7635](https://jira.percona.com/browse/PMM-7635): Investigate grafana-server high CPU consumption
```

Comment:
could you please align both high CPU together one after another and they rightly should be on top as most critical issue we have this patch release for ?

Reply:
Will do.